### PR TITLE
Revert "Tools: Topology2: Add IIR, FIR, DRC to HDA analog playback"

### PR DESCRIPTION
**Pull request by fredoh9**
This is reverting SOF PR8934.
https://github.com/thesofproject/sof/pull/8934

This reverts commit 727ad8299780edba92c61b53f50047fa3a6aa82f.

### CODE REVIEW

**Comment by singalsu** [comment]
@fredoh9  Can you try this instead. LNL platform is no more enabling components in app/boards/intel_adsp_ace20_lnl.conf

```
diff --git a/src/audio/drc/Kconfig b/src/audio/drc/Kconfig
index 9e6aaf638..90295bb1e 100644
--- a/src/audio/drc/Kconfig
+++ b/src/audio/drc/Kconfig
@@ -9,7 +9,7 @@ config COMP_DRC
        select NUMBERS_NORM
        select MATH_EXP
        select COMP_BLOB
-       default n
+       default y
        help
          Select for Dynamic Range Compressor (DRC) component. A DRC can be used
          to reduce the volume of loud sounds and amplify silent sounds thus
```

Another way would be to add there for DRC and others

CONFIG_COMP_DRC=y

@lrgirdwo @kv2019i Which approach would you recommend?

File: tools/topology/topology2/cavs-mixin-mixout-hda.conf [hunk 1]
```diff
@@ -7,9 +7,6 @@ Define {
 	DEEP_BUFFER_PIPELINE_SRC	'mixin.15.1'
 	DEEP_BUFFER_PIPELINE_SINK	'mixout.2.1'
 	DEEP_BUFFER_PCM_NAME		'Deepbuffer HDA Analog'
-	EFX_FIR_PARAMS			'passthrough'
```
Comment (singalsu) [inline tools/topology/topology2/cavs-mixin-mixout-hda.conf:10]: There is another fix

Comment (singalsu) [reply tools/topology/topology2/cavs-mixin-mixout-hda.conf:10]: See https://github.com/thesofproject/sof/pull/8963

**Review by kv2019i — APPROVED** [review]
I'll add my +1 one to this as well. I'd prefer https://github.com/thesofproject/sof/pull/8963 , but we should get one of these merged ASAP.

**Comment by kv2019i** [comment]
https://github.com/thesofproject/sof/pull/8963 merged, @fredoh9 please close this one if the tests are good with main.

**Comment by fredoh9** [comment]
Today's daily test looks good.

**Pull request closed without merging**